### PR TITLE
Port verbiage from explainer to Bikeshed file

### PR DESCRIPTION
**Pull request by melanierichards**
This PR ports a bunch of language from the explainer to the spec file. It mostly ports this verbiage as-is, so there are quite a few sections marked as non-normative, or which have meta-issues to formalize the language.

Notable changes from the explainer:

* Condensed the introduction
* Omitted open issues and feedback, which we will track via Github issues

### CODE REVIEW

File: is-logged-in.bs [hunk 1]
```diff
@@ -18,18 +18,184 @@ Level: None
...
+ISSUE: Update "website" with a more precise term
+
+<h3 id="setLoggedIn">Logged-in signal: <dfn export method for=Navigator><code>setLoggedIn()</code></dfn></h3>
+
+ISSUE: Flesh out section with explanatory text, an example, more formal/algorithmic language.
+
+```
+Navigator.setLoggedIn(
+    username: non-whitespace string of limited length,
+    credentialTokenType: “httpStateToken” OR “legacyAuthCookie”,
```
Comment (johnwilander) [inline is-logged-in.bs:54]: We've received https://github.com/privacycg/is-logged-in/issues/8 which asks that we don't refer to cookies as legacy (yet). I can only assume that what we call them will be what we get for the future too.

File: is-logged-in.bs [hunk 2]
```diff
@@ -18,18 +18,184 @@ Level: None
...
+
+<h3 id="isLoggedIn">Checking for a logged-in signal: <dfn export method for=Navigator><code>isLoggedIn()</code></dfn></h3>
+
+ISSUE: Flesh out section with explanatory text, an example, more formal/algorithmic language.
+
+```
+Navigator.isLoggedIn() –> Promise<bool>
+```
+
+<h2 id="defending-Against-abuse">Defending Against Abuse</h2>
```
Comment (johnwilander) [inline is-logged-in.bs:87]: Did you mean to capitalize "Against" in the id?

**Review by johnwilander — APPROVED** [review]
LGTM.

File: is-logged-in.bs (hunk 2 above)
Comment (melanierichards) [reply is-logged-in.bs:87]: Nope! Thanks John.

**Pull request merged**